Topic Wise Questions with Solutions/Linked List/remove.py

```diff
@@ -0,0 +1,42 @@
+"""
+Remove Element
+Given a linked list nums and a value val, remove all instances of that value in-place and return the new linked list.
+Do not allocate extra space for another linked list, you must do this by modifying the input linked list in-place with O(1) extra memory.
+Iterate the linked list and jump the values that needs to be deleted (change the next pointer).
+    Time Complexity:    O(N)
+    Space Complexity:   O(1)
+
+Constraints:
+
+0 <= nums.length <= 100
+0 <= nums[i] <= 50
+0 <= val <= 100
+
+
+"""
+print("Enter the values for nums to store in an array:")
```

Comment:
**Enter the values for nums to store in an array:**
change to **Enter the values for nums to store in an linkedist**